src/simulation/m_time_steppers.fpp

```diff
@@ -915,6 +938,133 @@ contains
 
     end subroutine s_adaptive_dt_bubble
 
+    subroutine s_compute_dt()
```

Comment:
should this not call a separate subroutine called `s_compute_cfl()`? if so, then we could add a feature where instead of specifying the time step size you specify the CFL and it computes step size for you (even without using adaptive time step sizes).

Reply:
That's exactly what this subroutine does. Calculates a time-step given a cfl number

Reply:
This can be accomplished by calling this subroutine once at the beginning of the simulation instead of at every time-step, but yes.

Reply:
This can be accomplished by calling this subroutine once at the first time step. I'll add it.

Reply:
indeed